packages/@react-types/calendar/src/index.d.ts

```diff
@@ -69,18 +69,30 @@ export interface AriaCalendarProps<T extends DateValue> extends CalendarProps<T>
 
 export interface AriaRangeCalendarProps<T extends DateValue> extends RangeCalendarProps<T>, DOMProps, AriaLabelingProps {}
 
+export type PageBehavior = 'single' | 'visible';
+
 export interface SpectrumCalendarProps<T extends DateValue> extends AriaCalendarProps<T>, StyleProps {
   /**
    * The number of months to display at once. Up to 3 months are supported.
    * @default 1
    */
-  visibleMonths?: number
+  visibleMonths?: number,
+  /**
+   * Controls the behavior of paging. Pagination either works by advancing the visible page by visibleDuration (default) or one unit of visibleDuration.
+   * @default visible
+   */
+  pageBehavior?: PageBehavior
```

Comment:
Could we move this type into CalendarPropsBase or something instead since it isn't spectrum specific